apps/backend/api/models/Group.js

```diff
@@ -1043,7 +1045,7 @@ module.exports = bookshelf.Model.extend(merge({
   async doesMenuUpdate ({ groupIds, post, customView, groupRelation = false }) {
     if (!post && !customView && !groupRelation) return
     console.log('broken here post', post)
```

Comment:
remove console.log (unless this is still draft then nvm :) )

Reply:
there is actually still a bit more work to do here, but whats here is reviewable. thanks

Reply:
Gotcha. One workflow that I like is to keep things in draft, and if we're doing a in process review then we still can request review and the draft status conveys this. when its not in draft I'm reviewing as if its going to production. come along with me on not that if its helpful, if not no thing and I'll adapt :)  

It's up for me now as I am thinking a PR process will be pretty helpful to keeping some clarity in the coming weeks.

Reply:
yeah I agree, should keep things as drafts. In this case I felt like the work done was reviewable and so I didnt keep it as a draft. the other work will just add on top